src/plugins/notebook/components/NotebookEmbed.vue

```diff
@@ -143,7 +143,9 @@ export default {
                 this.openmct.notifications.alert(message);
             }
 
-            window.location.hash = hash;
+            const relativeHash = hash.slice(hash.indexOf('#'));
```

Comment:
older version of notebook used to save this as full url, so need to extract only hash part for backwards compatibility.